app/components/peopletable/peopletable.js

```diff
@@ -131,6 +143,7 @@ const PeopleTable = translate()(class PeopleTable extends React.Component {
       this.setState({
         searching: false,
         dataList: this.buildDataList(),
+        showSearchReset: false,
```

Comment:
Missing: `searchPattern: ''` in the state.
If there is only one char in the search box, and remove it with backspace, it can't be removed.